presto-docs/src/main/sphinx/release/release-0.220.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+* Fix an issue when executing spatial join on top of a right join. See :issue:`12672`.
+* Fix an issue where ``TRY`` cannot catch the failures when ``ROW`` and ``ARRAY`` with ``NULL`` elements fail to compare.
+* Lazily build the hash table for map.
+
+Raptor Changes
+--------------
+* Integrate the optimized ORC writer into Raptor. Add configuration property ``storage.orc.optimized-writer-stage`` to enable and configure the optimized writer.
+  To switch to the optimized ORC writer is only backward compatible but not forward compatible.
+  Turning on the optimized ORC writer should work on an existing Raptor server. However, switching it back to Hadoop ORC writer can fail Raptor background job.
+* Integrate ``ZSTD`` compression for file writer. Add configuration property ``storage.orc.compression-kind`` to switch between ``SNAPPY`` and ``ZSTD``.
+  This is only applied when optimized ORC writer is turned on.
+
+
+Hive Connector Changes
+----------------------
+* Add ``compression_codec`` session property to set the compression codec when writing files. Possible values are ``NONE``, ``SNAPPY``, and ``GZIP``.
+* Add experimental support for partial merge pushdown, which optimizes plan for queries joining over two tables with mismatched but compatible bucket counts.
+  This feature can be enabled by session property ``experimental.optimizer.partial-merge-pushdown-strategy``, and is aiming to supersede ``hive.optimize-mismatched-bucket-count``.
+  See :issue:`12611`.
+* Improve writing to bucketed table by cache Hive Metastore.
+
+
+MongoDB Connector
+---------------------
+* Add ``mongodb.read-preference-tags`` configuration property to configure mongodb read preference tags.
+  Tag sets are separated by an ampersand, and each tag set is specified as a comma-separated list of colon-separated key-value pairs.
+  For example, ``mongodb.read-preference-tags=dc:east,use:reporting&use:reporting``.
+
+JDBC Driver Changes
+-------------------
+* Fix incorrect precision and column display size in ResultSetMetaData for char and varchar columns.
+
+Verifier Changes
+----------------
+* Fix an issue where Verifier always mark floating point columns as mismatched if one of the control checksum and the test checksum is 0 while the other is close to 0.
```

Comment:
Fix an issue where Verifier would always mark floating point columns as mismatched if either the control checksum or test checksum was 0 while the other was close to 0.